src/redux/actions/brownbagActions.js

```diff
@@ -1,36 +1,50 @@
 import fetch from 'isomorphic-fetch';
+import { normalize, schema } from 'normalizr';
 import * as actions from './actionTypes';
 import fetchUrl from '../../config';
 
-export function requestUnpresentedUsers(user) {
+
+var brownbags = new schema.Entity('brownbags');
```

Comment:
Can we use `let` or `const` instead of var here, for ES6 uniformity

Reply:
Let me change that, I forgot to change after debugging.

Reply:
Cool, also check a couple of missing semi colons in this file

Reply:
Cool, fix it right away